src/components/Checkbox/index.tsx

```diff
@@ -73,6 +73,7 @@ type CheckboxProps = {
 
 const Checkbox: FunctionComponent<CheckboxProps> = ({
   checked = false,
+  state,
```

Comment:
I didn't found any other way to do this, because `checkbox.state` is generated using the `checked` prop, not the `state` one. However we can get the `state` prop with `props.state` instead of destructuring the props. Should I implement this second solution?

Reply:
It seems to be possible to set the indeterminate state externally https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes

```
var checkbox = document.getElementById("some-checkbox");
checkbox.indeterminate = true;
```

Reply:
Ok nevermind, just saw that react use the same trick, 👌 for me :)